python/tools/wlr-libpy/README.md

```diff
@@ -0,0 +1,48 @@
+# About
+
+Helper crate for linking to the pre-built libpython from Webassembly Language Runtimes.
+
+# Features
+
+## build
+
+This feature is intended for usage in `build.rs` scripts. It will download all needed pre-built static libraries for `wasm32-wasi` and configure the linker to use them.
+
+To use this feature add this to your Cargo.toml
+
+```toml
+[build-dependencies]
+wlr-libpy = { git = "https://github.com/vmware-labs/webassembly-language-runtimes.git", branch="rust-py-example", features = ["build"] }
+```
+
+Then, in the [build.rs](./build.rs) file of your project you only need to call this:
+
+```rs
+    use wlr_libpy::bld_cfg::configure_static_libs;
+    configure_static_libs().unwrap().emit_link_flags();
+```
```

Comment:
I would wrap these methods in the `main` function directly. Most projects don't have a `build.rs` and people are not used to code them.